loom/graphics/gfxVectorRenderer.cpp

```diff
@@ -296,17 +296,16 @@ void VectorRenderer::textBox(float x, float y, float width, utString* string) {
 }
 
 Loom2D::Rectangle VectorRenderer::textLineBounds(VectorTextFormat* format, float x, float y, utString* string) {
-    float* bounds = new float[4];
+    float bounds[4];
     nvgSave(nvg);
     nvgReset(nvg);
     textFormat(format);
-    nvgTextBounds(nvg, x, y, string->c_str(), NULL, bounds);
+    nvgTextBounds(nvg, x, y, string->c_str(), NULL, &bounds[0]);

```

Comment:
Can this stay as bounds? I thought array was the same as &array[0] (but maybe in some cases it is not... I seem to remember exception(s)).